sparkles/core.py

```diff
@@ -1351,6 +1353,25 @@ def check_include_exclude(self):
 
                     self.add_message("info", msg)
 
+    def check_t_ccd_acq(self):
+        """
+        Check that the fid lights are in the current search boxes by a check on t_ccd_acq.
+        These tolerances apply to observations after the 2022:294 safe mode.
+        """
+        if not self.is_ER and CxoTime(self.date).date > "2022:294":
+            if self.fids.t_ccd_acq <= -14.0:
```

Comment:
Since you use `self.fids.t_ccd_acq` four times, define a short-cut like `t_ccd_acq` at this point and use it. Don't Repeat Yourself!